wom/src/main/scala/wom/types/WomBigDecimalType.scala

```diff
@@ -0,0 +1,64 @@
+package wom.types
+
+import spray.json.{JsNumber, JsString}
+import wom.values.{WomBigDecimal, WomInteger, WomLong, WomString}
+
+import scala.util.{Success, Try}
+
+case object WomBigDecimalType extends WomPrimitiveType {
+  val stableName: String = "BigDecimal"
+
+  override protected def coercion = {
+    case i: Integer => WomBigDecimal(BigDecimal(i))
```

Comment:
unnecessary explicit wrapping

Reply:
Somewhy there is no implicit convertation from `Integer` to `BigDecimal`, therefore I leave this as is.